traffic_monitor_golang/traffic_monitor/enum/enum.go

```diff
@@ -32,6 +32,9 @@ import (
 	"strings"
 )
 
+// TrafficMonitorType is the Traffic Ops 'type' of Traffic Monitor servers. This is case-sensitive.
+const TrafficMonitorType = "RASCAL"
```

Comment:
I think this should be a config. Someone could setup their CDN with a different type, or we could potentially use a different type in the future.